Procfile

```diff
@@ -1 +1 @@
-web: uvicorn consumatio.external.api:app --host=0.0.0.0 --port=${PORT:-5000}
+web: python3 api.py --host=0.0.0.0 --port=${PORT:-5000}
```

Comment:
`api.py` isn't at the root directory - also, shouldn't `--host` and `--port` be dropped? Additionally, please set the `HOST` variable here (`HOST=0.0.0.0 python3`), Heroku only sets `PORT`.

Reply:
See following commit.